docs/RUNNING-LOCALLY.md

```diff
@@ -4,6 +4,8 @@
 > This describes command line operations in the immediate "local" context,
 > without using Kubernetes and not running the application in Docker.
 
+> Note: If using a custom connection string or custom app settings in your .NET application, it's highly recommended to add an `appsettings.Development.json` file for your local development environment. This file should be based on the `appsettings.json` file and can override settings specifically for local development.
```

Comment:
Please let the reader know _where_ (directory) to create that file.

Proper format for a GitHub style note is:

```
> [!NOTE]
> Add your text here
```

For example:

> [!Note]
> Add your text here